content/docs/security.md

```diff
@@ -0,0 +1,91 @@
+
+See an example of [generating self-signed certificates](https://github.com/pingcap/docs/blob/master/op-guide/generate-self-signed-certificates.md) using `cfssl`.
+
+### Configure the TiKV Server Certificates
+
+Specify the TLS options via command line or the configuration file.
+
+The relevant options in the configuration file are:
+
+```toml
+# Using empty strings here means disabling secure connections.
+[security]
+# The path to the file that contains the PEM encoding of the server’s CA certificates.
+ca-path = "/path/to/ca.pem"
+# The path to the file that contains the PEM encoding of the server’s certificate chain.
+cert-path = "/path/to/tikv-server-cert.pem"
+# The path to the file that contains the PEM encoding of the server’s private key.
+key-path = "/path/to/tikv-server-key.pem"
+```
+
+You'll also need to **change the corresponding URL to `https://`**.
+
+### Configure the Placement Driver Certificates
+
+```toml
+[security]
+# The path to the file that contains the PEM encoding of the server’s CA certificates.
+cacert-path = "/path/to/ca.pem"
+# The path to the file that contains the PEM encoding of the server’s certificate chain.
+cert-path = "/path/to/pd-server-cert.pem"
+# The path to the file that contains the PEM encoding of the server’s private key.
+key-path = "/path/to/pd-server-key.pem"
+```
+
+You'll also need to **change the corresponding URL to `https://`**.
+
+### Configure the Client
+
+When connecting your TiKV Client, you'll need to specify the TLS options. In this example we build a configuration for the [Rust Client](https://github.com/tikv/client-rust):
+
+```rust
+let config = Config::new(/* ... */).with_security(
+    // The path to the file that contains the PEM encoding of the server’s CA certificates.
+    "/path/to/ca.pem",
+    // The path to the file that contains the PEM encoding of the server’s certificate chain.
+    "/path/to/tikv-client-cert.pem",
+    // The path to the file that contains the PEM encoding of the server’s private key.
+    "/path/to/tikv-client-key.pem"
+);
+```
+
+You'll also need to **change the corresponding URL to `https://`**.
+
+## On-Disk Encryption
+
+TiKV currently does not offer built-in on disk encryption. If this is a requirement for your deployment you can consider creating an encrypted volume.
+
+This feature is part of the planned [roadmap](https://github.com/tikv/tikv/blob/master/docs/ROADMAP.md#engine) under 'Pluggable Engine Interface'. *(See this [Issue #3680](https://github.com/tikv/tikv/issues/3680) if you want to help.)*
+
+## Reporting Vulnerabilities
+
+For virtually all vulnerabilities you are invited to open a ['Bug Report'](https://github.com/tikv/tikv/issues/new?template=bug-report.md) on our issue tracker.
+
+For situations where the vulnerability must be kept secret to maintain data security or integrity, you can contact [a maintainer](https://github.com/tikv/tikv/blob/master/MAINTAINERS.md). They are best equipped to handle these critical situations.

```

Comment:
```suggestion
For situations where the vulnerability must be kept secret to maintain data security or integrity, you can contact our [maintainers](https://github.com/tikv/tikv/blob/master/MAINTAINERS.md), who are best equipped to handle these critical situations.
```

Reply:
Keeping it "A maintainer" to discourage folks from mailing everyone.